src/PrestaShopBundle/Resources/views/Admin/Configure/ShopParameters/maintenance.html.twig

```diff
@@ -0,0 +1,93 @@
+ * International Registered Trademark & Property of PrestaShop SA
+ *#}
+{% extends '@PrestaShop/Admin/layout.html.twig' %}
+{% trans_default_domain "Admin.Shopparameters.Feature" %}
+
+{%
+    set generalForm = form.general
+%}
+
+{% block content %}
+    <div class="container">
+        {{ form_start(form, {'attr' : {'class': 'form'} }) }}
+        <div class="row">
+            {% block maintenance_form_general %}
+            <div class="col">
+                <div class="card">
+                    <h3 class="card-header">
+                        <i class="material-icons">business_center</i> {{ 'General'|trans }}
+                    </h3>
+                    <div class="card-block">
+                        <div class="card-text">
+                            <div class="form-group">
+                                {{ ps.label_with_help(('Enable Shop'|trans), ('Activate or deactivate your shop (It is a good idea to deactivate your shop while you perform maintenance. Please note that the webservice will not be disabled).'|trans({}, 'Admin.Shopparameters.Help'))) }}
+                                {{ form_errors(generalForm.disable_shop) }}
+                                {{ form_widget(generalForm.disable_shop) }}
+                            </div>
+                            <div class="form-group">
+                                {{ ps.label_with_help(('Maintenance IP'|trans), ('IP addresses allowed to access the front office even if the shop is disabled. Please use a comma to separate them (e.g. 42.24.4.2,127.0.0.1,99.98.97.96)'|trans({}, 'Admin.Shopparameters.Help'))) }}
+                                {{ form_errors(generalForm.maintenance_ip) }}
+                                {{ form_widget(generalForm.maintenance_ip) }}
+                            </div>
+                            <div class="form-group">
+                                {{ ps.label_with_help(('Custom maintenance text'|trans), ('Custom text displayed on maintenance page while shop is deactivated.'|trans({}, 'Admin.Shopparameters.Help'))) }}
+                                {{ form_errors(generalForm.maintenance_text) }}
+                                {{ form_widget(generalForm.maintenance_text) }}
+                            </div>
+                        </div>
+                    </div>
+                    <div class="card-footer">
+                        <button class="btn btn-primary">{{ 'Save'|trans({}, 'Admin.Actions') }}</button>
+                    </div>
+                </div>
+            </div>
+            {% endblock %}
+
+        </div>
+        {{ form_end(form) }}
+    </div>
+{% endblock %}
+
+{% block javascripts %}
+    {{ parent() }}
+    <script src="{{ asset('../js/tiny_mce/tiny_mce.js') }}"></script>
+    <script src="{{ asset('../js/admin/tinymce.inc.js') }}"></script>
+    <script src="{{ asset('../js/admin/tinymce_loader.js') }}"></script>
+    <script>
```

Comment:
maybe extract this script into MaintenancePage and get it using Webpack? you can reuse the strategy used here => https://github.com/PrestaShop/PrestaShop/pull/8523/commits/844ee7b3845956c1738b5ac6c7fafffd396caee6

The long terme idea: everything - including tiny MCE - will be loaded using Webpack :+1: